.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -4,25 +4,16 @@ about: Create a bug report to help us improve
 title: ''
 labels: bug
 assignees: ''
-
 ---
 
 <!-- Please answer these questions when reporting a bug. Thanks! -->
 
-### What version of iotex-core are you using (`ioctl version`)?
```

Comment:
`ioctl version` not accurate, it gives the version of ioctl binary installed on local machine, but not the current running image/code

Reply:
Ok